src/ismn/components.py

```diff
@@ -467,10 +467,14 @@ def get_sensors(self, variable, depth_from, depth_to):
         sensors : numpy.ndarray
             array of sensors found for the given combination of variable and depths
         """
-        return np.array([
-            s for s in self.iter_sensors(
-                variable=variable, depth=Depth(depth_from, depth_to))
-        ])
+        return np.array(
```

Comment:
this looks like black, can you apply `yapf file.py --in-place` to all files where you changed anything?